inst/include/mode/mode.hpp

```diff
@@ -17,22 +17,42 @@ class container {
   using model_type = T;
   using pars_type = mode::pars_type<T>;
 
-  container(const pars_type& pars, const double time) {
+  container(const pars_type& pars, const double time,
+            const size_t n_particles): n_particles_(n_particles) {
     // This will get considerably more complicated once we move to
     // support multiple particles, and following the general approach
     // in dust will be sensible, I think.
     auto m = model_type(pars);
     auto y = m.initial(time);
     auto ctl = control();
-    solver_.push_back(solver<model_type>(m, time, y, ctl));
+    for (size_t i; i < n_particles; ++i){
```

Comment:
```suggestion
    for (size_t i = 0; i < n_particles; ++i) {
```

amazingly this does actually work, but it does cause a compiler warning for me:

```
   In file included from /home/rich/Documents/src/mode/inst/include/mode/r/mode.hpp:10,
                    from mode.cpp:1:
   /home/rich/Documents/src/mode/inst/include/mode/mode.hpp: In constructor ‘mode::container<T>::container(const pars_type&, double, size_t) [with T = logistic]’:
   /home/rich/Documents/src/mode/inst/include/mode/mode.hpp:28:22: warning: ‘i’ may be used uninitialized in this function [-Wmaybe-uninitialized]
      28 |     for (size_t i; i < n_particles; ++i){
         |                    ~~^~~~~~~~~~~~~
```

(plus trivial whitespace issue)